Задания с 25.02.2022/Task2/Task2/PostfixCalculator.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+
+namespace PostfixCalculator;
+
+/// <summary>
+/// Реализация постфиксного калькулятора на выбранном стеке
+/// </summary>
+public class PostfixCalculator
+{
+    private IStack stack;
+
+    public PostfixCalculator(IStack stack)
+    {
+        this.stack = stack;
+    }
+
+    /// <summary>
+    /// Подсчитывает значение по постфиксной записи
+    /// </summary>
+    /// <param name="expression">Постфиксная запись</param>
+    public (double?, bool) CalculateExpression(string expression)
+    {
+        string[] tokens = expression.Split(' ');
+        bool isCorrectSequence = true;
+
+        foreach(var token in tokens)
+        {
+            if (!isCorrectSequence)
+            {
+                break;
+            }
+
+            int number;
+            bool isNumber = int.TryParse(token, out number);
```

Comment:
```suggestion
            bool isNumber = int.TryParse(token, out int number);
```
Вообще, переменная без значения --- это всегда странно